### ISSUE TITLE
Unable to make a new report from the reports screen

### ISSUE THREAD

**Issue by wrightmartin** [mysociety/fixmystreet#1751]
If you're browsing a local council, there's no way to make a new report on the map. You have to click 'new report' in the header which puts you right back at the beginning of the process, you have to enter your postcode etc from scratch.

To me this feels unintuitive, and there's a use-case here for adding a new report here. You could use this screen to check if a fault you've spotted has a report or not

![image](https://user-images.githubusercontent.com/2292925/27953555-1cd5d0d8-6304-11e7-8580-0bb37b8ce338.png)

 https://www.fixmystreet.com/reports/Shropshire

**Comment by dracos** [comment]
Yes, there's no real reason this couldn't act like an around page now (it was more complicated in the past); even report pages (if you come from an around page) let you do so with JS on.

**Comment by dracos** [comment]
If you zoom in a bit the "report a problem" becomes "Report a problem here" and does let you start immediately, though not exactly same as /around.
